libraryManagement/settings.py

```diff
@@ -1,145 +0,0 @@
-"""
-Django settings for libraryManagement project.
-
-Generated by 'django-admin startproject' using Django 5.0.6.
-
-For more information on this file, see
-https://docs.djangoproject.com/en/5.0/topics/settings/
-
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/5.0/ref/settings/
-"""
-
-from pathlib import Path
-
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
-
-
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
-
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-!h+kd#o*&lj$_1z@awk&jb6@blq%fjbg41(csnd^x8^k$(s--)"
-
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
-
-ALLOWED_HOSTS = []
-
-
-# Application definition
-
-INSTALLED_APPS = [
-    "django.contrib.admin",
-    "django.contrib.auth",
-    "django.contrib.contenttypes",
-    "django.contrib.sessions",
-    "django.contrib.messages",
-    "django.contrib.staticfiles",
-    "books",
```

Comment:
separate all the apps like:
INSTALLED_APPS = [
    "django.contrib.admin",...]
CUSTOM_APPS = ["authors", ...]
THIRDPARTY_APPS = [...]
INSTALLED_APPS +=CUSTOM_APPS + THIRDPARTY_APPS